spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/flyway/FlywayProperties.java

```diff
@@ -328,6 +329,33 @@
 	 */
 	private Boolean skipExecutingMigrations;
 
+	/**
+	 * The REST API URL of the Conjur server. Requires Flyway teams.
```

Comment:
Configuration properties documentation don't start with "The" or "A". Please look at other description in this file (and others) for inspiration and [this section of the reference guide](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-custom-starter-configuration-keys).